### DISCUSSION TITLE
Please add support for DeepSeek V2.5

### DISCUSSION THREAD

**Discussion by Jasmin68k** [cline/cline#657] [Feature Requests]
Hello!

See title, it's a good model, would be great to have it available:

https://www.deepseek.com/

Thank you!

**Comment by heltonteixeira** [comment]
You can use openai compatible option and use:
Base URL: https/api.deepseek.com/beta
Model: deepseek-chat

**Comment by Jasmin68k** [comment]
Awesome, thanks!
